src/pydata_sphinx_theme/assets/styles/content/_math.scss

```diff
@@ -0,0 +1,16 @@
+/**
+ * hack the display of equations to make sure they don't overlap on mobile display
+ */
+.math {
+  align-items: center;
+  display: flex;
```

Comment:
```suggestion
  display: flex;
  flex-direction: row-reverse;
  gap: .5em;
```

Reply:
flex tricks will never ceased to amaze me